lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3054,57 +3054,75 @@ namespace {
       if (!clangDecl->hasAttrs())
         return;
 
-      SmallVector<ValueDecl *, 1> results;
-      auto conformsToAttr =
-          llvm::find_if(clangDecl->getAttrs(), [](auto *attr) {
-            if (auto swiftAttr = dyn_cast<clang::SwiftAttrAttr>(attr))
-              return swiftAttr->getAttribute().starts_with("conforms_to:");
-            return false;
-          });
-      if (conformsToAttr == clangDecl->getAttrs().end())
-        return;
+      DeclAttribute *baseAttrsSentry = *decl->getAttrs().begin();
```

Comment:
Having a long-lived iterator like this makes me worried about possible iterator invalidation. Can we call `getAttrs().add(...)` to add the attributes instead?

Reply:
Do you mean insert a sentry into attrs list?